src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private static final String type = "square";
+    private final int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) side * side;
+    }
+
+    @Override
+    public String getInfo() {
+        return "Figure: "
+                + type + ", "
+                + "area: "
+                + this.getArea()
+                + " sq. units, "
+                + "side: "
+                + side
+                + " units, "
+                + "color: "
+                + color;
```

Comment:
Use the 'name()' method instead of 'toString()' for getting the String representation of enum constants. If 'color' is an enum, it should be 'color.name();' otherwise, if 'color' is a String, this is fine.